vic/drivers/classic/src/read_soilparam.c

```diff
@@ -692,6 +696,108 @@ read_soilparam(FILE *soilparam,
             temp.Ws = temp.Ws / temp.max_moist[layer];
         }
 
+        // Soil thermal node thicknesses and positions
```

Comment:
Yes, I think you are right.  Good catch.  I need to move those into both read_soilparam() and vic_init().